doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,164 +3,239 @@
 
-Workaround: Manually update the value of the controller URL property in the {EAP} `standalone.xml` and  `standalone-full.xml` file.
+.If a case task actor is invalid, the resulting BPMN file is also invalid [https://issues.jboss.org/browse/JBPM-8603[JBPM-8603]]
 
+Issue: The new process designer sometimes generates an invalid XML file . A user task with an actor assigned is generated as a a `potentialOwner` element in the XSD file.
 
-== {KIE_SERVER}
+[source]
+----
+<bpmn2:potentialOwner id="5058b718-8866-4ccd-b793-d5ebf48de5a2">
+<bpmn2:resourceAssignmentExpression id="_gN_feI4_Eem2-an8Fwu06w">
+<bpmn2:formalExpression id="_gN_feY4_Eem2-an8Fwu06w">manager</bpmn2:formalExpression>
+</bpmn2:resourceAssignmentExpression>
+</bpmn2:potentialOwner>
+----
 
-.You cannot deploy a KJAR with a PMML asset to a {KIE_SERVER} Spring Boot starter on Windows [https://issues.jboss.org/browse/RHDM-852[RHDM-852]]
+The id of the `potentialOwner` element is missing an underscore at the beginning, which means that if the first element is a number, this is now an invalid ID.
 
-Issue: If a KJAR contains a PMML asset, you cannot deploy it as a container on a {KIE_SERVER} that is running in the `kie-server-spring-boot-starter` Spring Boot starter on Microsoft Windows.
+Tasks that have `ioSpecification` should always define an `inputSet` and `outputSet`, even if those are empty.
 
 Steps to reproduce:
 
-. Create a KJAR that contains a PMML asset.
-. Start `kie-server-spring-boot-starter` on Windows.
-. Deploy the KJAR to a {KIE_SERVER} that is running the `kie-server-spring-boot-starter` Spring Boot starter on Microsoft Windows.
+Create a user task with no output mapping. This will generate the following output:
 
-Expected result: The KJAR is deployed successfully.
+[source]
+----
+<bpmn2:ioSpecification id="_rEnOQY5AEem2-an8Fwu06w">
+<bpmn2:dataInput id="38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_TaskNameInputX" drools:dtype="Object" itemSubjectRef="_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_TaskNameInputXItem" name="TaskName"/>
+<bpmn2:dataInput id="38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_SkippableInputX" drools:dtype="Object" itemSubjectRef="_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_SkippableInputXItem" name="Skippable"/>
+<bpmn2:inputSet id="_rEnOQo5AEem2-an8Fwu06w">
+<bpmn2:dataInputRefs>_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_TaskNameInputX</bpmn2:dataInputRefs>
+<bpmn2:dataInputRefs>_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_SkippableInputX</bpmn2:dataInputRefs>
+</bpmn2:inputSet>
+</bpmn2:ioSpecification>
+----
 
-Actual result: The KJAR cannot be deployed.
+Expected result: The output includes an empty `outputSet` attribute.
+
+Actual result: The output does not include an `outputSet` attribute.
 
 Workaround: None.
 
-ifdef::DM[]
-== OpenShift
 
-.User password is displayed in the Readiness and Liveness probes [https://issues.jboss.org/browse/RHDM-848[RHDM-848]]
+.Service Task is not usable at runtime [https://issues.jboss.org/browse/JBPM-8581[JBPM-8581]]
```

Comment:
Hey @emmurphy1 
This issue is not anymore relevant for the RHPAM 7.4.x release, because finally we're not able to include the service task on this release, it will be available in the next 7.5 one instead, it's actually only present on the community versions, but not yet for RHPAM. 
As this service task type has been finally not included in the RHPAM distributions (see [RHPAM-2183](https://issues.jboss.org/browse/RHPAM-2183)) I think it's better not including it on the documentation.
Thx!

Reply:
Thanks, removed.